var/spack/repos/builtin/packages/py-markdown/package.py

```diff
@@ -36,7 +37,9 @@ class PyMarkdown(PythonPackage):
     depends_on("python@2.7:2.8,3.2:3.6", when="@2.6.8:2.6.11")
     depends_on("python@2.7:2.8,3.3.5:", when="@3.1.1:")
     depends_on("python@3.6:", when="@3.3.4:")
+    depends_on("python@3.7:", when="@3.4.1:")
 
-    depends_on("py-setuptools", type="build", when="@2.6.11:")
-    depends_on("py-setuptools@36.6:", type="build", when="@3.1:")
-    depends_on("py-importlib-metadata", type=("build", "run"), when="@3.3.4: ^python@:3.7")
+    depends_on("py-setuptools", when="@2.6.11:", type="build")
+    depends_on("py-setuptools@36.6:", when="@3.1:", type="build")
+    depends_on("py-importlib-metadata", when="@3.3.4: ^python@:3.7", type=("build", "run"))
```

Comment:
Thanks for reordering the args.  Reads better.

Reply:
@tldahlgren I can do a batch update of all packages (it's a simple `sed`) if everyone agrees and if it goes into a single PR...